JetMETCorrections/Configuration/python/JetCorrectors_cff.py

```diff
@@ -48,7 +47,16 @@
     algorithm   = cms.string('AK4PFchs'),
     srcRho      = cms.InputTag( 'fixedGridRhoFastjetAll' )
     )
-ak4JPTL1FastjetCorrector = ak4CaloL1FastjetCorrector.clone()
+#### JPT L1Fastjet need L1CaloFastjet
+ak4L1JPTFastjetCorrector = cms.EDProducer(
+    'L1JPTOffsetCorrectorProducer',
+    level = cms.string('L1JPTOffset'),
+    algorithm = cms.string('AK4JPT'),
+    offsetService = cms.InputTag('ak4CaloL1FastjetCorrector')
+    )
+ak4L1JPTFastjetCorrectorChain = cms.Sequence(
+    ak4CaloL1FastjetCorrector * ak4L1JPTFastjetCorrector
+)
```

Comment:
@kodolova 

I am wondering if this sequence is really used somewhere?
"ak4L1JPTFastjetCorrectorChain"

Regards,
Taejeong